content/src/view.js

```diff
@@ -3176,6 +3281,165 @@ function setupHpanelBox(box) {
   });
 }
 
+function arrow(pane, arrow_list, traceEventNum, show_fade) {
+  /* note: we expect arrow_list to be an array of key value pairs where 
+  each key is a color for the arrow, and each value is a list of location pairs
+  to draw an arrow on.   */
+
+  var dropletEditor = state.pane[pane].dropletEditor;
+  var arrow_data = arrow_list[traceEventNum];
```

Comment:
Hmm, the comment above says that the arrows are keyed on color, but this is indexing them by some kind of number. (and I haven't found arrow color anywhere yet). Is the comment stale? what is the arrow_data data structure actually like? is that explicitly commented anywhere? :D